beta-prompts/beta_prompts/templating_framework.py

```
@@ -0,0 +1,1029 @@
+        template_data = self.registry.get_template(template_id)
+        if not template_data:
+            raise ValueError(f"Template {template_id} not found")
+
+        if format == "json":
+            return json.dumps(template_data, indent=2)
+        if format == "yaml":
+            return yaml.dump(template_data, default_flow_style=False)
+        raise ValueError(f"Unsupported export format: {format}")
+
+    def import_template(
+        self, template_data: Union[str, dict], format: str = "json"
```

Ruff (0.12.2)

Function argument `format` is shadowing a Python builtin

(A002)